.github/workflows/publish.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         if: inputs.run_id <= 0
         with:
           node-version: 16
-      - run: npm install @actions/artifact
+      - run: npm install @actions/artifact@1
```

Comment:
Minor: this job doesn't use our manylinux Docker image, so it should be able to run on Node 20.

You can try removing lines 20-23 instead.